src/core/crypto/nacl_catapult.ts

```diff
@@ -667,21 +667,21 @@ const unpack25519 = (o, n) => {
 };
 
 export const cleanup = (arr) => {
-    for (var i = 0; i < arr.length; i++) {
+    for (let i = 0; i < arr.length; i++) {
         arr[i] = 0;
     }
 };
 
-export const crypto_shared_key_hash = (shared, pk, sk, hashfunc) => {
+export const crypto_shared_key_hash = (shared, pk, sk, hashfunc, hasherType) => {
```

Comment:
hasherType is actually `signSchema` ?

Reply:
That's correct, but I don't think we used it anymore, but I will change it anyway, in case it get picked up by the wallet codes.